### ISSUE TITLE
[BUG] XEMM cannot start. Error parsing symbol xzcbtc: 'NoneTYpe' object has no attribute 'group'

### ISSUE THREAD

**Issue by sweetpotatoz** [CoinAlpha/hummingbot#1033]
**Describe the bug**
// A clear and concise description of what the bug is.

XEMM cannot start. Error parsing symbol xzcbtc: 'NoneTYpe' object has no attribute 'group'
But trading is set as XZC-BTC during config.

I have stop and start the bot.  stop the container, rm it, and create it again.  Also tried delete the folder, and recreate everything as well.

Log as below, when I try to start the XEMM bot
```
2019-11-05 12:50:16,947 - 10 - hummingbot.data_feed.coin_gecko_data_feed - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2019-11-05 12:50:17,711 - 10 - hummingbot.data_feed.coin_cap_data_feed - INFO - Network status has changed to NetworkStatus.CONNECTED. Starting networking...
2019-11-05 12:51:00,028 - 10 - root - ERROR - Unhandled error in background task: list.remove(x): x not in list
Traceback (most recent call last):
  File "/hummingbot/core/utils/async_utils.py", line 8, in safe_wrapper
    return await c
  File "/hummingbot/client/command/stop_command.py", line 32, in stop_loop
    self.clock.remove_iterator(self.strategy)
  File "clock.pyx", line 83, in hummingbot.c

```
**Steps To Reproduce**
// A concise description of the **steps to reproduce the buggy behavior**:
use the attached strategy config file.  import and start
cross exchange, maker=huobi, taker=binance, trading pair=XZC-BTC

Here are the steps to reproduce the issue (see attachments in the section below):
1. Set up your configuration file with these parameters: ...
2. Start the bot using this command: ...
...

**Screenshots**
// If applicable, add screenshots to help explain your problem.
![xemm_xzcbtc](https://user-images.githubusercontent.com/1692346/68209782-1d0b2300-000f-11ea-8156-52f418820aa1.png)
![xemm_xzcbtc2](https://user-images.githubusercontent.com/1692346/68209784-1d0b2300-000f-11ea-8a98-9848b87f0252.png)

**Release version**
// Include your bot's version number (Can be found at the bottom left corner of your CLI).
0.18.0

**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 
The strategy file .yml renamed to txt

[xemm_binancehuobi_xzcbtc.txt](https://github.com/CoinAlpha/hummingbot/files/3809055/xemm_binancehuobi_xzcbtc.txt)

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
**Optional: your ETH address:** (If you want to participate in the bug bounty)

**Comment by rennel-tabing** [comment]
Same issue has been reported by user bob_tan using `BAT-USDT` pair using maker exchange: `huobi` ; taker exchange: `binance` and I was able to reproduce the error.

The issue maybe related to the changes on cross-exchange token pair from previous version `batusdt` to `BAT-USDT`  though this pair is working just fine on PureMM strategy.

**Comment by Clint-G13** [comment]
Re-run configuration using a different pair and the same error occurred on both latest version and latest development commit.

                "branch": "development",
                "commit": "e79902b6a2b3105e9e8811a635ee16df1a16edda",
                "date": "2019-11-04T23:38:46Z",
                "maintainer": "CoinAlpha, Inc. <dev@coinalpha.com>"
![bug#1033(dev)](https://user-images.githubusercontent.com/57189990/68237753-bc95d900-0042-11ea-95f6-bce96b38326b.PNG)

**Comment by PtrckM** [comment]
fix on dev-0.19.0

```
                "branch": "development",
                "commit": "5577a37c4907e86eb8b8366b301b0d08338274c8",
                "date": "2019-11-06T04:29:29Z",
```

![image](https://user-images.githubusercontent.com/54516858/68270512-6c496600-0098-11ea-991e-554abdb503ae.png)

**Comment by sweetpotatoz** [comment]
updated to dev-0.19.0
issue resolved.
